website/docs/docs/dbt-versions/release-notes/09-April-2023/scheduler-optimized.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "dbt Cloud more intelligently processes scheduled jobs"
+id: "scheduler-optimized"
+description: "April 2023: "
+sidebar_label: "Update: Scheduler optimizes job queue"
+tags: [Apr-12-2023, scheduler]
+---
+
+The dbt Cloud Scheduler now prevents queue clog by canceling queued runs in excess of 50. The job queue can build up when the duration of a job run grows over time, usually caused by more data landing in your cloud warehouse. Once the run duration becomes longer than the frequency of the job’s schedule, runs start to queue, waiting for the long running job to finish. 
```

Comment:
qq - is it dbt Cloud Scheduler or dbt Cloud scheduler? suggesting lowercase for consistency 

```suggestion
The dbt Cloud scheduler now prevents queue clog by canceling queued runs in excess of 50. The job queue can build up when the duration of a job run grows over time, usually caused by more data landing in your cloud warehouse. Once the run duration becomes longer than the frequency of the job’s schedule, runs start to queue, waiting for the long-running job to finish. 
```